conf/modules.config

```diff
@@ -614,21 +614,27 @@ if (params.macs_gsize) {
         }
     }
 
-    // if (!params.skip_diff_analysis) {
-    //     process {
-    //         withName: DESEQ2_FEATURECOUNTS {
-    //             publishDir = [
-    //                 path: { [
-    //                     "${params.outdir}/${params.aligner}/mergedLibrary/macs2",
-    //                     params.narrow_peak? '/narrowPeak' : '/broadPeak',
-    //                     '/consensus/deseq2'
-    //                     ].join('') },
-    //                 mode: 'copy',
-    //                 saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-    //             ]
-    //         }
-    //     }
-    // }
+    if (!params.skip_diff_analysis) {
```

Comment:
We aren't doing any differential analysis now?

Reply:
Yeap, you are right will drop the parameter for the moment from everywhere...